src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -102,6 +107,18 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
     programmeSlug,
   } = commonPathway;
 
+  const isLegacyLicense = !lessonCohort || lessonCohort === LEGACY_COHORT;
+  const isNew = lessonCohort === NEW_COHORT;
+
+  const isMathJaxLesson =
+    subjectSlug &&
+    ALLOWED_MATHJAX_SUBJECT_SLUGS.includes(subjectSlug) &&
+    !isLegacyLicense
+      ? hasLessonMathJax(lesson)
```

Comment:
can hasLessonMathjax do all these checks internally? would be cleaner than having the component have to know about all these conditions

Reply:
Yes could do, would need to pass lesson, subjectSlug and isLegacyLicense and do the ALLOWED_MATHJAX_SUBJECT_SLUGS check in the function. is that what you mean ?

Reply:
yup exactly, can just extract this exact piece of code into another function